rngd.8.in

```diff
@@ -196,6 +196,18 @@ source is disabled by default as it should never be used for cryptographic
 purposes, or any use case in which random data should be known only by a single
 entity.
 
+.TP
+.B
+Qrypt Entropy Beacon (qrypt) [Index 9]
+Entropy gathered from the qrypt entropy beacon.  Note this beacon works in a
+fashion to the NIST beacon, fetching remotely generated entropy and delivering
+it over a network.  Like NIST, this should not be used for cryptographic
```

Comment:
Instead of "Like NIST, ..." since the Qrypt API doesn't provide the entropy in the same schema as NIST or keep records of pulses as NIST beacon does, it might be better to not even mention NIST. A more appropriate wording here may be: "This source should not be used for cryptographic operations unless the network can be guaranteed secure against man-in-the-middle attacks."